ddl/index.go

```diff
@@ -684,9 +683,11 @@ func (w *addIndexWorker) handleBackfillTask(d *ddlCtx, task *reorgIndexTask) *ad
 	result := &addIndexResult{addedCount: 0, nextHandle: handleRange.startHandle, err: nil}
 	lastLogCount := 0
 	startTime := time.Now()
+	index := tables.NewIndex(task.partitionID, w.table.Meta(), w.indexInfo)
```

Comment:
task.partitionID confuses me, we need to do a refactor after this PR.

Reply:
@shenli is doing that.